post-microsservice/src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
+spring.jpa.database=mysql
+spring.datasource.url = jdbc:mysql://localhost:3306/posts
+spring.datasource.username = root
```

Comment:
Should't username and password be hidden from code?

Reply:
Yes. But we need to create an application file to prod. We'll make it when dockerfile is ready.